src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -565,37 +551,47 @@ private async void SynchronizationContext_UnhandledException(object sender, Unha
         private async Task ReportUnhandledException(Exception exception)
         {
             Log.CircuitUnhandledException(_logger, CircuitId, exception);
+
             if (!Client.Connected)
             {
-                _logger.LogDebug("Client is disconnected YO.");
+                Log.UnhandledExceptionClientDisconnected(_logger, CircuitId, exception);
```

Comment:
I would expect this to appear at L582 so that it gets logged in all cases.

Reply:
done